src/components/views/settings/tabs/room/SecurityRoomSettingsTab.js

```diff
@@ -83,14 +85,36 @@ export default class SecurityRoomSettingsTab extends React.Component {
     };
 
     _onEncryptionChange = (e) => {
-        const beforeEncrypted = this.state.encrypted;
-        this.setState({encrypted: true});
-        MatrixClientPeg.get().sendStateEvent(
-            this.props.roomId, "m.room.encryption",
-            { algorithm: "m.megolm.v1.aes-sha2" },
-        ).catch((e) => {
-            console.error(e);
-            this.setState({encrypted: beforeEncrypted});
+        Modal.createTrackedDialog('Enable encryption', '', QuestionDialog, {
+            title: _t('Enable encryption?'),
+            description: _t(
+                "Once enabled, encryption for a room cannot be disabled. Messages sent in an encrypted " +
+                "room cannot be seen by the server, only be the participants of the room. Enabling encryption " +
```

Comment:
```suggestion
                "room cannot be seen by the server, only by the participants of the room. Enabling encryption " +
```

(also in `src/i18n/strings/en_EN.json`)

Reply:
(applied manually because multiple files - good catch)